### PR TITLE
Add link to Tails website

### PR DESCRIPTION
**Pull request by apyrgio**
Point users to the installation instructions of Dangerzone in the Tails website. These instructions were recently added to Tails, and we have worked with the Tails developers to make this integration happen.

See also:
* https://tails.net/news/dangerzone/index.en.html
* https://gitlab.tails.boum.org/tails/tails/-/issues/20355

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -253,6 +254,12 @@ column to "Selected".
 You can now launch Dangerzone from the list of applications for your qube, and
 pass it a file to sanitize.
 
+## Tails
+
+Dangerzone is not yet available by default in Tails, but we have collaborated
+with the Tails team to offer some manual
+[installation instructions](https://tails.net/doc/persistent_storage/additional_software/dangerzone/index.en.html)
+to Tails users.
```
Comment (legoktm) [inline INSTALL.md:262]: nit:
```suggestion
for Tails users.
```

File: INSTALL.md [hunk 2]
```diff
@@ -253,6 +254,12 @@ column to "Selected".
 You can now launch Dangerzone from the list of applications for your qube, and
 pass it a file to sanitize.
 
+## Tails
+
+Dangerzone is not yet available by default in Tails, but we have collaborated
+with the Tails team to offer some manual
```
Comment (legoktm) [inline INSTALL.md:260]: nit:
```suggestion
with the Tails team to offer manual
```

**Review by legoktm — APPROVED** [review]
Minor grammar suggestions, but otherwise LGTM

**Comment by apyrgio** [comment]
Merged with the suggested fixes:

```diff
diff --git a/INSTALL.md b/INSTALL.md                                                                                   
index 76a3b5e..3aeaf06 100644                                                                                          
--- a/INSTALL.md                                                                                                       
+++ b/INSTALL.md                                                                                                       
@@ -257,9 +257,9 @@ pass it a file to sanitize.      
 ## Tails                                                                                                              
                                                                                                                       
 Dangerzone is not yet available by default in Tails, but we have collaborated       
-with the Tails team to offer some manual                                                                              
+with the Tails team to offer manual                                                                                   
 [installation instructions](https://tails.net/doc/persistent_storage/additional_software/dangerzone/index.en.html)    
-to Tails users.                                                                                                       
+for Tails users.                                                                                                      
                                                                                                                       
 ## Build from source  
 ```
 
 Thanks for the review Kunal!

**Pull request merged**